Extensions/PrimitiveDrawing/shapepainterruntimeobject.ts

```diff
@@ -337,6 +345,26 @@ namespace gdjs {
       return this._clearBetweenFrames;
     }
 
+    setAntialiasingOn(value: boolean): void {
+      this._antialiasing = value;
+    }
+    
+    isAntialiasingOn(): boolean {
+      return this._antialiasing;
+    }
+
+    setAntialiasingQuality(value: integer): void {
+      if(value !== 2 && value !== 4 && value !== 8) {
+        //by default, assume we want the lowest quality to avoid crashing the game.
```

Comment:
Add a space before and capitalize your comments. I think it can also be worded more precisely:
```suggestion
        // If an invalid value has been passed, default to the lowest quality, as overuse of higher quality anti-aliasing may slow down or crash the game.
```